Makefile

```diff
@@ -9,8 +9,7 @@ ifeq (${uname},Darwin)
 	OS=darwin
 endif
 
-utils = github.com/goreleaser/goreleaser \
-		github.com/golang/dep/cmd/dep
+export GO111MODULE=on
```

Comment:
I'd like to suggest using module mirror by adding `GOPROXY=https://proxy.golang.org`
See: https://proxy.golang.org/

Reply:
Thanks for suggetion!
Module mirror seems fine, so I'll adopt it!

2b21a3a